pkg/ingester/ingester.go

```diff
@@ -221,6 +228,10 @@ func New(cfg Config, chunkStore ChunkStore) (*Ingester, error) {
 			Name: "cortex_ingester_queried_samples_total",
 			Help: "The total number of samples returned from queries.",
 		}),
+		queriedSeries: prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
I think this would be more useful as a histogram? What do you think?

Reply:
Yes. But, that also applied to the `queriedSamples` metric too right? If yes, I'll go ahead and make those changes.

Reply:
Histogram would be great. Thanks!